src/components/footer/index.njk

```diff
@@ -117,7 +145,35 @@
         text: 'No'
       },
       {
-        text: 'Array of items to display in the list'
+        text: 'Array of items to display in the list in navigation section of the footer.'
+      }
+    ],
+    [
+      {
+        text: 'navigation.items.{}.text'
+      },
+      {
+        text: 'string'
+      },
+      {
+        text: 'No'
+      },
+      {
+        text: 'List item text in the navigation section of the footer.'
+      }
+    ],
+    [
+      {
+        text: 'navigation.items.{}.href'
+      },
+      {
+        text: 'string'
+      },
+      {
+        text: 'No'
+      },
+      {
+        text: 'List item href atrribute in the navigation section of the footer. Both `text` and `href` attributes need to be present to create a link.'
```

Comment:
Nit: typo in "attribute".

Reply:
can't see the typo

Reply:
It's in "atrribute" - sorry should have said that.